### PR TITLE
adding 2 icons

### PR DESCRIPTION
**Pull request by asafstr2**

### CODE REVIEW

**Review by seanmcintyre — CHANGES_REQUESTED** [review]
Hi @asafstr2! Please format these to follow the conventions of other Iris icons. You can view any other icon as an example: https://github.com/vimeo/iris/blob/main/src/icons/ui/Camera.tsx.

- The `forwardRef` HOC must be used.
- The only attribtues on the `svg` element should be `viewBox`, `ref`, and spread `props`.
- If possible, remove `fill` attributes from child elements of `svg`. It's not required, but it is preferred if it is not a necessary value.

File: src/icons/ui/Create.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React, { forwardRef, Ref, SVGAttributes } from 'react';
+
+export const Create = forwardRef(
+    (props: SVGAttributes<SVGSVGElement>, ref: Ref<SVGSVGElement>) => (
+        <svg
+            width="24"
+            height="24"
```
Comment (seanmcintyre) [inline src/icons/ui/Create.tsx:7]: Please remove.

File: src/icons/ui/Create.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            viewBox="0 0 24 24"
+            fill="none"
```
Comment (seanmcintyre) [inline src/icons/ui/Create.tsx:9]: is this attribute necessary?

File: src/icons/ui/Create.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+            xmlns="http://www.w3.org/2000/svg"
```
Comment (seanmcintyre) [inline src/icons/ui/Create.tsx:10]: Please remove.

File: src/icons/ui/Theme.tsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
+import React, { forwardRef, Ref, SVGAttributes } from 'react';
+
+export const Theme = forwardRef(
+    (props: SVGAttributes<SVGSVGElement>, ref: Ref<SVGSVGElement>) => (
+        <svg
+            width="24"
+            height="24"
```
Comment (seanmcintyre) [inline src/icons/ui/Theme.tsx:7]: Please remove.

File: src/icons/ui/Theme.tsx [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+            viewBox="0 0 24 24"
+            fill="none"
```
Comment (seanmcintyre) [inline src/icons/ui/Theme.tsx:9]: is this attribute necessary?

File: src/icons/ui/Theme.tsx [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+            xmlns="http://www.w3.org/2000/svg"
```
Comment (seanmcintyre) [inline src/icons/ui/Theme.tsx:10]: Please remove.

File: src/icons/ui/Theme.tsx [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+            ref={ref} {...props}
+                >
+            <g id="Icon/Dark/Theme">
```
Comment (seanmcintyre) [inline src/icons/ui/Theme.tsx:13]: Is this tag necessary?

File: src/icons/ui/Theme.tsx [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+                <rect
+                    id="Rectangle"
+                    opacity="0.01"
```
Comment (seanmcintyre) [inline src/icons/ui/Theme.tsx:16]: Why does this this icon need to manually style a child element with SVG attributes?

File: src/icons/ui/Theme.tsx [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+            width="24"
+            height="24"
+            viewBox="0 0 24 24"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+            ref={ref} {...props}
+                >
+            <g id="Icon/Dark/Theme">
+                <rect
+                    id="Rectangle"
```
Comment (seanmcintyre) [inline src/icons/ui/Theme.tsx:15]: Please remove.

File: src/icons/ui/Theme.tsx [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+                    opacity="0.01"
+                    width="24"
+                    height="24"
+                />
+                <g id="photo_filter-24px">
```
Comment (seanmcintyre) [inline src/icons/ui/Theme.tsx:20]: Is this tag necessary? Also, please remove the `id` attribute.

File: src/icons/ui/Theme.tsx [hunk 11]
```diff
@@ -0,0 +1,31 @@
...
+                    <path
+                        id="Shape"
```
Comment (seanmcintyre) [inline src/icons/ui/Theme.tsx:22]: Please remove.

**Review by seanmcintyre — CHANGES_REQUESTED** [review]
Please run prettier on your files.

File: src/icons/ui/Theme.tsx (hunk 11 above)
Comment (asafstr2) [reply src/icons/ui/Theme.tsx]: Removed

File: src/icons/ui/Theme.tsx (hunk 10 above)
Comment (asafstr2) [reply src/icons/ui/Theme.tsx]: Fixed

File: src/icons/ui/Theme.tsx (hunk 9 above)
Comment (asafstr2) [reply src/icons/ui/Theme.tsx]: Fixed

File: src/icons/ui/Theme.tsx (hunk 8 above)
Comment (asafstr2) [reply src/icons/ui/Theme.tsx]: Fixed

File: src/icons/ui/Theme.tsx (hunk 7 above)
Comment (asafstr2) [reply src/icons/ui/Theme.tsx]: Fixed

File: src/icons/ui/Theme.tsx (hunk 5 above)
Comment (asafstr2) [reply src/icons/ui/Theme.tsx]: Fixed

File: src/icons/ui/Theme.tsx (hunk 4 above)
Comment (asafstr2) [reply src/icons/ui/Theme.tsx]: Fixed

File: src/icons/ui/Create.tsx (hunk 3 above)
Comment (asafstr2) [reply src/icons/ui/Create.tsx]: Removed

File: src/icons/ui/Create.tsx (hunk 2 above)
Comment (asafstr2) [reply src/icons/ui/Create.tsx]: Fixed

File: src/icons/ui/Create.tsx (hunk 1 above)
Comment (asafstr2) [reply src/icons/ui/Create.tsx]: Removed

**Review by seanmcintyre — CHANGES_REQUESTED** [review]
Please run prettier over the added files to address the failing build.

**Pull request merged**